### PR TITLE
Add example using LB 3.0

### PR DESCRIPTION
**Pull request by Amir-61**
* Add example using LB 3.0

@strongloop/squad-epitome @superkhau @bajtos PTAL. 

Please note we have two folders under this repo called `example` and `example-2.0` which use LB 1.x and LB 2.x accordingly; for consistency I created a separate folder for LB3.0 called `example-3.0`.

Connect to https://github.com/strongloop-internal/scrum-loopback/issues/1189

### CODE REVIEW

**Comment by pthieu** [comment]
What's up with the `angular-file-upload.min.js` minified file? Is this needed? I guess it's a scaffolding file.

I'm assuming all these files are auto-generated files so user can jump to a certain step. In that case, LGTM.

File: example-3.0/server/server.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var boot = require('loopback-boot');
+var compression =  require('compression');
+var loopback = require('loopback');
+
+var app = module.exports = loopback();
+
+app.start = function() {
+  // start the web server
+  // request pre-processing middleware
+  app.use(compression());
```
Comment (bajtos) [inline example-3.0/server/server.js]: This should go to `server/middleware.json`.

**Review by bajtos — CHANGES_REQUESTED** [review]
Looks mostly good, see one comment below.

**Comment by Amir-61** [comment]
@bajtos Addressed the issue. PTAL again Thanks!

**Review by bajtos — APPROVED** [review]
LGTM.

Please consider adding https://github.com/strongloop/loopback-workspace/blob/27572451152fdcadf15de6f3ab4b11316a3f27a0/templates/projects/empty-server/files/server/middleware.development.json to enable useful error responses & logs in development.

**Comment by Amir-61** [comment]
> Please consider adding https://github.com/strongloop/loopback-workspace/blob/27572451152fdcadf15de6f3ab4b11316a3f27a0/templates/projects/empty-server/files/server/middleware.development.json to enable useful error responses & logs in development.

Thanks I added your suggestion as well 👍 ... I'm going to land it now

**Pull request merged**